techniques/android/MASTG-TECH-0112.md

```diff
@@ -0,0 +1,89 @@
+|title    |platform    |
+|---------|------------|
+|Reverse Engineering Flutter Applications | android |
+
+Flutter is an open-source UI SDK by Google for building natively

```

Comment:
Please do not break lines within the same paragraph. Each paragraph should be in one line (represented by one line number)